bindings/core/src/lib.rs

```diff
@@ -42,14 +43,21 @@ pub fn init_logger(config: String) -> std::result::Result<(), fern_logger::Error
 #[derivative(Debug)]
 #[serde(rename_all = "camelCase")]
 pub struct WalletOptions {
-    pub storage_path: Option<String>,
-    pub client_options: Option<ClientOptions>,
+    pub address: Option<Bech32Address>,
+    pub alias: Option<String>,
     pub bip_path: Option<Bip44>,
+    pub client_options: Option<ClientOptions>,
     #[derivative(Debug(format_with = "OmittedDebug::omitted_fmt"))]
     pub secret_manager: Option<SecretManagerDto>,
+    pub storage_path: Option<String>,
 }
 
 impl WalletOptions {
+    pub fn with_address(mut self, address: impl Into<Option<Bech32Address>>) -> Self {
+        self.address = address.into();
+        self
+    }
+
```

Comment:
why no `with_alias`?

Reply:
also ... a bit nit-pickish ... but can you align the method order with the field order?

Reply:
Wondering if we should remove them all, they're not used apart from tests
But added and aligned